src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -1,5 +1,168 @@
 import React from 'react';
 import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import Str from 'expensify-common/lib/str';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
 
-const ValidationStep = () => <View />;
-export default ValidationStep;
+import {validateBankAccount} from '../../libs/actions/BankAccounts';
+
+import Button from '../../components/Button';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import TextInputWithLabel from '../../components/TextInputWithLabel';
+import Text from '../../components/Text';
+import BankAccount from '../../libs/models/BankAccount';
+import CONST from '../../CONST';
+import ImageView from '../../components/ImageView';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    achData: PropTypes.shape({
+        bankAccountID: PropTypes.number.isRequired,
+        state: PropTypes.string,
+    }),
```

Comment:
I think maybe `achData` would need to be required as well in this case since it's not possible to require a field on an object that is not required 🤔